File: fizzbuzz/fizzbuzz.c
#include <stdio.h>

void fizzbuzz(int n) {
    for (int i = 1; i <= n; i++) {
        if (i % 3 == 0 && i % 5 == 0) {
            printf("FizzBuzz\n");
        } else if (i % 3 == 0) {
            printf("Fizz\n");
        } else if (i % 5 == 0) {
            printf("Buzz\n");
        } else {
            printf("%d\n", i);
        }
    }
}

int main() {
    int n;

    printf("Enter a number to count until: ");
    if (scanf("%d", &n) == 1) {
        fizzbuzz(n);
    } else {
        printf("Please enter a valid integer.\n");
    }

    return 0;
}

File: fizzbuzz/fizzbuzz.py
#!/usr/bin/env python3

def fizzbuzz(n):
    for i in range(1, n + 1):
        if i % 3 == 0 and i % 5 == 0:
            print("FizzBuzz")
        elif i % 3 == 0:
            print("Fizz")
        elif i % 5 == 0:
            print("Buzz")
        else:
            print(i)

try:
    n = int(input("Enter a number to count until: "))
    fizzbuzz(n)
except ValueError:
    print("Please enter a valid integer.")
